owners/src/ownership/schema.json

```diff
@@ -29,15 +29,15 @@
   "definitions": {
     "Bot": {
       "type": "string",
-      "pattern": "^[a-zA-Z\\d][a-zA-Z\\d-_.]{2,37}[a-zA-Z\\d]\\[bot\\]$"
+      "pattern": "^[a-zA-Z\\d][a-zA-Z\\d_.-]{2,37}[a-zA-Z\\d]\\[bot\\]$"
     },
     "User": {
       "type": "string",
-      "pattern": "^[a-zA-Z\\d][a-zA-Z\\d-_.]{2,37}[a-zA-Z\\d]$"
+      "pattern": "^[a-zA-Z\\d][a-zA-Z\\d_.-]{2,37}[a-zA-Z\\d]$"
```

Comment:
Why did the order of these have to change?

Reply:
I'm glad you asked cause it took me like 2 hours to figure it out and I *have* to tell someone! 😂

When JS parses a string to a RegExp with the `u` (unicode flag) it assumes that any character can be part of a range inside a `[…]` block - so it was trying to find the "range" between `\d` and `.` which is not a thing (because `\d` is itself a range)

ajv started parsing string RegExp with the `u` flag, so we have to put the `-` character last

Reply:
![image](https://user-images.githubusercontent.com/6694512/132743788-e927127e-7168-45a8-ad3e-3acb67de44fa.png)